src/request-data/resource.js

```diff
@@ -51,7 +51,7 @@ class BaseResource {
   }
 }
 
-const _container = Symbol('container');
+export const _container = Symbol('container');
```

Comment:
`_container` is supposed to work like a private property, so it'd be great to avoid exporting it.

Reply:
One last thing, would you mind removing `export` on this line as well?